_includes/commands/save-cmds.md

```diff
@@ -0,0 +1,12 @@
+
+| Command   | Description                                                                                                                    |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| mceCancel | Resets the editor content to the last save point created with `mceSave`, the _save_ toolbar button, or the **Save** menu item. |
```

Comment:
Not sure if you're worried but it'll also reset back to the initial state if nothing has been saved.

Reply:
fixed